compiler/rustc_resolve/src/build_reduced_graph.rs

```diff
@@ -190,10 +204,17 @@ impl<'a, 'tcx> Resolver<'a, 'tcx> {
     }
 
     pub(crate) fn build_reduced_graph_external(&mut self, module: Module<'a>) {
-        for child in self.tcx.module_children(module.def_id()) {
+        let def_id = module.def_id();
+        for child in self.tcx.module_children(def_id) {
             let parent_scope = ParentScope::module(module, self);
             BuildReducedGraphVisitor { r: self, parent_scope }
-                .build_reduced_graph_for_external_crate_res(child);
+                .build_reduced_graph_for_external_crate_res(child, false);
+        }
+        let children = self.cstore().ambiguity_module_children_untracked(def_id, self.tcx.sess);
+        for child in &children {
+            let parent_scope = ParentScope::module(module, self);
```

Comment:
This line is common for both loops and can be moved out of them, if borrow checker allows.